myget.bat

```diff
@@ -0,0 +1,25 @@
+REM This is all ugly... we'll want to revisit this as soon as possible
+call dnvm use 1.0.0-rc1-update1 -r clr
+
+REM Temporary measure while we resolve oddities with the local feed
+REM cache and version numbers. Basically, we want to fetch these from
+REM nuget.org.
+rmdir /s /q c:\Users\Wonka\.dnx\packages\Google.Apis
+rmdir /s /q c:\Users\Wonka\.dnx\packages\Google.Apis.Core
+rmdir /s /q c:\Users\Wonka\.dnx\packages\Google.Apis.Auth
+
+call dnu restore
```

Comment:
I suspect that would be at the cost of simple diagnostics. It's easier to find out what's wrong with one thing happening at a time... and this definitely isn't performance critical.